themes/default/gulpfile.js

```diff
@@ -221,6 +236,8 @@ gulp.task('make-js', ['make-js-components', 'make-js-npm'], function() {
 		.pipe(gulp.dest('js'))
 });
 
+gulp.task('build', ['json', 'pure', 'make-css', 'cms-css', 'make-js', 'svgo']);
```

Comment:
Can we also just update watch to use build here: 

    gulp.task('watch', ['build', 'browserSync'], function () {

This part only gets run when starting gulp, not every time, so it shouldn't slow things down.